app/models/runtime/process_model.rb

```diff
@@ -576,9 +576,7 @@ def permitted_users
     end
 
     def docker_run_action_user
-      return AppModel::DEFAULT_CONTAINER_USER unless docker?
```

Comment:
I _think_ this ends up being a duplicate check if docker_run_action_user only gets called after the model has checked that the process belongs to a docker lifecycle app.